src/main.rs

```diff
@@ -130,12 +132,29 @@ async fn main() -> ExitCode {
     }
 }
 
-#[instrument]
+#[instrument(name = "root")]
 async fn provision(config: Config, opts: Cli) -> Result<(), anyhow::Error> {
+    let system = System::new();
+    let kernel_version = system
+        .kernel_version()
+        .unwrap_or("Unknown Kernel Version".to_string());
+    let os_version = system
+        .os_version()
+        .unwrap_or("Unknown OS Version".to_string());
+    let azure_init_version = env!("CARGO_PKG_VERSION");
+
+    tracing::info!(
+        "Kernel Version: {}, OS Version: {}, Azure-Init Version: {}",
+        kernel_version,
+        os_version,
+        azure_init_version
```

Comment:
We capture the version in a constant so you could replace this with that:
```suggestion
        VERSION
```